xml/ha_migration.xml

```diff
@@ -647,6 +647,22 @@
      <para>taroth 2015-10-06: as we continue, add more listitems here...</para> 
     </listitem-->
    </itemizedlist>
+
+   <para>Use one the following procedures for your scenario:</para>
```

Comment:
one of? :-)

Reply:
Yes, you are right. :-) Fixed in b2cba13d.